### ISSUE TITLE
Manual Asset Upload documentation doesn't specify which attributes are optional and which are required

### ISSUE THREAD

**Issue by gregorydlogan** [opencast/opencast#1115]
Per https://opencast.jira.com/browse/MH-12915

The table on the Manual Asset Upload documentation does not indicate which attributes are required and which are optional (displayFallback & displayOverride are optional attributes)

https://docs.opencast.org/r/5.x/admin/configuration/asset-upload-ui/#step-2-add-your-new-option-to-the-list-configuration

**Comment by whoisvishesh** [comment]
@gregorydlogan is this issue is still active? Can I proceed with this?

**Comment by gregorydlogan** [comment]
@whoisvishesh Yep, still active.  If you're working on this make sure you're working on the new admin ui in develop or r/14.x, since the current ui is going to go away sooner than later.

**Comment by Arnei** [comment]
Fixed by #7833
